packages/schema/lib/schemas/FieldSchema.js

```diff
@@ -19,192 +13,132 @@ const formatBullet = (f) => `* ${f.map(wrapInBackticks).join(' & ')}`;
+        'The type of this value. Use `string` for basic text input, `text` for a large, `<textarea>` style box, and `code` for a `<textarea>` with a fixed-width font. Field type of `file` will accept either a file object or a string. If a URL is provided in the string, Zapier will automatically make a GET for that file. Otherwise, a .txt file will be generated.',
+      type: 'string',
+      // string == unicode
+      // text == a long textarea string
+      // integer == int
+      // number == float
+      enum: [
+        'string',
+        'text',
+        'integer',
+        'number',
+        'boolean',
+        'datetime',
+        'file',
+        'password',
+        'copy',
+        'code',
+      ],
+    },
+    required: {
+      description: 'If this value is required or not.',
+      type: 'boolean',
+    },
+    default: {
+      description:
+        'A default value that is saved the first time a Zap is created.',
+      type: 'string',
+      minLength: 1,
+    },
+    list: {
+      description:
+        'Acts differently when used in inputFields vs. when used in outputFields. In inputFields: Can a user provide multiples of this field? In outputFields: Does this field return an array of items of type `type`?',
+      type: 'boolean',
+    },
+    children: {
+      type: 'array',
+      items: { $ref: '/FieldSchema' },
+      description:
+        'An array of child fields that define the structure of a sub-object for this field. Usually used for line items.',
+      minItems: 1,
+    },
+    dict: {
+      description: 'Is this field a key/value input?',
+      type: 'boolean',
+    },
   },
-  [RefResourceSchema, FieldChoicesSchema, FieldMetaSchema],
-);
+  examples: [
+    // 1. Minimal valid example
+    { key: 'abc' },
+
+    // 2. Has a label
+    { key: 'abc_label', label: 'First Name' },
+
+    // 3. Has a type and required
+    { key: 'abc_required', type: 'boolean', required: true },
+
+    // 4. Has a default
+    { key: 'abc_default', default: 'some default' },
+
+    // 5. Children array referencing FieldSchema
+    { key: 'parent', children: [{ key: 'child' }] },
+
+    // 6. A field with type=integer
+    { key: 'abc_int', type: 'integer' },
+  ],
+  antiExamples: [
+    {
+      example: {},
+      reason: 'Missing required key: key',
+    },
+    {
+      example: { key: 'abc', type: 'loltype' },
+      reason: 'Invalid value for key: type',
+    },
+    {
+      // If someone tries to add a "choices" property, it's invalid
+      example: { key: 'abc', choices: { mobile: 'Mobile Phone' } },
+      reason: 'Invalid extra property: choices (not in schema)',
+    },
+    {
+      // default must be at least 1 character long
+      example: { key: 'abc', default: '' },
+      reason: 'Invalid value for key: default (cannot be empty string)',
+    },
+    {
+      // key must be at least 1 character long
+      example: { key: '' },
+      reason: 'Invalid value for key: key (cannot be empty string)',
+    },
```

Comment:
Since the FieldSchema is not actually inherited in each of the new 3 schemas, these examples and antiExamples wouldn't be used in any auto-validation for fields' values. I think it is safe to remove both.

Reply:
hmmm the [auto-validation for FieldSchema](https://cdn.zappy.app/f582cc546189a5cced52e07541486da7.gif) does seem to run the examples/anti-examples though, when removed it shows as "2 tests pending" for the schema

Reply:
Sounds good

Reply:
It seems you forgot to return them; they are missing.

Reply:
Yup added them back in now!